fhirlib/test/fhir/medreq-min-rDVch-expected.shexj

```diff
@@ -56,31 +56,31 @@
           },
           {
             "type": "TripleConstraint",
-            "predicate": "http://hl7.org/fhir/identifier",
+            "predicate": "http://hl7.org/fhir/CanonicalResource.identifier",
             "valueExpr": "http://hl7.org/fhir/shape/Identifier",
```

Comment:
I agree that these are legit "rDVch" changes, i.e. that predicates should contain Datatypes and Valuetypes, but I'm surprised that you stumbled across it. If you're simply fixing it 'cause you noticed it, +1, but if you're specifically targeting rDVch, keep in mind that it doesn't match the configuration of the last two versions:

| FHIR version | FHR/RDF config |
| -- | -- |
| R4 | RDVch |
| R5 | rdvCh |

Reply:
I figured it out: these are not deliberate changes that should be part of this PR, but -- since I worked on it on my macOS laptop, which has case-insensitive filenames, which means that the following two files collide on my filesystem:

```
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'fhirlib/test/fhir/medreq-min-RDVch-expected.shexj'
  'fhirlib/test/fhir/medreq-min-rDVch-expected.shexj'
```

Is there any way you could rename one of these files to avoid this collision? Alternatively I could recreate this PR on a Linux system at work, but I think this collision will cause problems to people using macOS or Windows in the future.